docs/requirements.txt

```diff
@@ -4,4 +4,4 @@ sphinx-tabs==3.2.0
 sphinx-rtd-theme==0.5.2
 sphinxemoji==0.1.8
 sphinx-autoapi==1.8.4
-sphinx-notfound-page
\ No newline at end of file
+sphinx-notfound-page==0.7.1
```

Comment:
```suggestion
# We need to ignore upgrading it here, because we install it from the
# repository when building the docs on Read the Docs
sphinx-notfound-page  # pyup: ignore
```